main.tf

```diff
@@ -57,15 +59,15 @@ module "vault_cluster" {
   # source = "github.com/hashicorp/terraform-aws-vault//modules/vault-cluster?ref=v0.0.1"
   source = "./modules/vault-cluster"
 
-  cluster_name  = "${var.vault_cluster_name}"
-  cluster_size  = "${var.vault_cluster_size}"
-  instance_type = "${var.vault_instance_type}"
+  cluster_name  = var.vault_cluster_name
+  cluster_size  = var.vault_cluster_size
+  instance_type = var.vault_instance_type
 
-  ami_id    = "${var.ami_id == "" ? data.aws_ami.vault_consul.image_id : var.ami_id}"
-  user_data = "${data.template_file.user_data_vault_cluster.rendered}"
+  ami_id    = var.ami_id == "" ? data.aws_ami.vault_consul.image_id : var.ami_id
```

Comment:
Shouldn't the default of `var.ami_id` be null now?

Reply:
46b245452069037aabba9d49d873fe6306c0c306